starport_template/lib/model/balance_json.dart

```diff
@@ -0,0 +1,18 @@
+class BalanceJson {
+  late String denom;
+  late String amount;
+
+  BalanceJson({required this.denom, required this.amount});
+
+  BalanceJson.fromJson(Map<String, dynamic> json) {
+    denom = json['denom'] as String;
+    amount = json['amount'] as String;
+  }
+
+  Map<String, dynamic> toJson() {
```

Comment:
we don't use `toJson` anywhere, lets remove it

Reply:
✅